presto-spi/src/main/java/com/facebook/presto/spi/block/DictionaryBlock.java

```diff
@@ -271,7 +271,7 @@ public long getPositionsSizeInBytes(boolean[] positions)
                 used[getId(i)] = true;
             }
         }
-        return dictionary.getPositionsSizeInBytes(used) + Integer.BYTES * countUsedPositions(positions);
+        return dictionary.getPositionsSizeInBytes(used) + (Integer.BYTES * (long) countUsedPositions(positions));
```

Comment:
the added parens are redundant

Reply:
They are redundant, in that they don't change the behavior, but I find it improves readability and serves to clarify the intention.